servicer/builtin/service_adapters/package/base_package.py

```diff
@@ -131,10 +129,19 @@ def if_package_version_exists(self, **package_info):
 
         return package_info['version_exists']
 
+    def get_existing_versions(self, **package_info):
+        if 'existing_versions_source' not in package_info:
+            raise NameError('Service must provide existing_versions_source.')
+        print('existing_versions_source is: %s' % package_info['existing_versions_source'])
```

Comment:
Use `self.logger.log('existing_versions_source is: %s' % package_info['existing_versions_source'])` over `print()`. It has options for controlled output levels.
For example, you can leave messages that only show up when `DEBUG=1` when servicer runs using this:
```
self.logger.log('this is a debug message', level='debug')
```